api/src/test/java/org/openmrs/module/patientqueueing/api/PatientQueueingServiceTest.java

```diff
@@ -33,6 +39,8 @@
 	
 	private static final String QUEUE_STANDARD_DATASET_XML = "org/openmrs/module/patientqueueing/standardTestDataset.xml";
 	
+	Logger logger = LoggerFactory.getLogger(PatientQueueingServiceTest.class);
```

Comment:
Did you just forget how we log? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-loggingLogging